packaging/suse/far2l.spec

```diff
@@ -21,7 +21,7 @@ Distribution: 	openSUSE Tumbleweed
 BuildRequires:  cmake
 BuildRequires:  gcc-c++
 BuildRequires:  pkgconfig
-BuildRequires:  gawk m4 make
```

Comment:
isn't awk must be removed here but not m4?

Reply:
.m4 files are parsed using far2l_m4 Perl script now.

Reply:
It does not require m4 itself.

Reply:
ok, but still is gawk should be listed in BuildRequires here?

Reply:
Now everything is done in Perl, neither awk nor m4 needed any more.

Reply:
fixed